### ISSUE TITLE
ingress issue

### ISSUE THREAD

**Issue by jr0dd** [minio/minio#13068]
I just wanted to report that when using the helm chart to deploy, it is assigning the main minio port and not the minio-console port for the ingress. One has to manually patch the ingress to get it to load the console.

**Comment by harshavardhana** [comment]
@jr0dd can you provide more information?  - we can fix it right away.

**Comment by jr0dd** [comment]
```

~ % kubectl get svc -n minio
NAME            TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
minio-console   ClusterIP   172.17.54.227    <none>        9001/TCP   7m8s
minio           ClusterIP   172.17.186.180   <none>        9000/TCP   7m8s
```

```

spec:
  ingressClassName: traefik
  rules:
  - host: minio.domain.tld
    http:
      paths:
      - backend:
          service:
            name: minio
            port:
              number: 9001
        path: /
        pathType: ImplementationSpecific
  tls:
  - hosts:
    - minio.domain.tld
    secretName: minio-tls
```
Here's some output from the services and ingress. The ingress is using the port `minio` instead of `minio-console`. This was configured under consoleIngress.

**Comment by jr0dd** [comment]
@harshavardhana Or I can work up a PR later on today. Whatever you prefer.

**Comment by harshavardhana** [comment]
```
 git diff
diff --git a/helm/minio/templates/console-ingress.yaml b/helm/minio/templates/console-ingress.yaml
index cb486e68f..f2858a95a 100644
--- a/helm/minio/templates/console-ingress.yaml
+++ b/helm/minio/templates/console-ingress.yaml
@@ -1,5 +1,5 @@
 {{- if .Values.consoleIngress.enabled -}}
-{{- $fullName := include "minio.fullname" . -}}
+{{- $fullName := printf "%s-console" (include "minio.fullname" .) -}}
 {{- $servicePort := .Values.consoleService.port -}}
 {{- $ingressPath := .Values.consoleIngress.path -}}
 apiVersion: {{ template "minio.consoleIngress.apiVersion" . }}
```

@jr0dd can you apply this and see if it fixes the problem?

**Comment by harshavardhana** [comment]
This is fixed in 6e5f83c45b4ffdebeca3658d26d0b663f3b4d5b6

**Comment by jr0dd** [comment]
> {{- $fullName := printf "%s-console" (include "minio.fullname" .) -}}

That works
